src/App.js

```diff
@@ -1,16 +1,96 @@
-import React from 'react';
+import React from "react";
+import Search from "./Search";
+import Results from "./Results";
+// import Info from "./Info";
+
+let movieArray = [];
 
 class App extends React.Component {
-  constructor(){
+  constructor(props) {
     super();
+    this.state = {
+      movies: [],
+      currentMovie: "",
+      currentInfo: "",
+      sortBy: ""
+
+    };
+    this.sortMovie = this.sortMovie.bind(this)
+    this.receiver = this.receiver.bind(this);
+    this.receiveMovie = this.receiveMovie.bind(this);
+    this.clearResults = this.clearResults.bind(this);
+  }
+
+  //receiver function to populate movies in state
+  receiver(results) {
+    movieArray.unshift(results);
+    this.setState({ movies: movieArray });
+    console.log(this.state.movies)
```

Comment:
avoid committing console.logs. Also, this console.log is likely to run before setState has had an effect as it is asynchronous. To console.log after setState has taken effect you can use

```js
  this.setState({ movies: movieArray }, () => console.log(this.state.movies));
```

Where the second parameter passed to setState is a callback which will be called after setState has completed.